docs/schema.md

```diff
@@ -752,9 +754,28 @@ is required for each `credits` entry.
 Each `credits[].contact[]` entry should be a valid, fully qualified, plain-text URL
 at which the credited can be reached. Providing contacts is optional.
 
+### credits[].type[] field
+
+The `credits[].type[]` field should specify type or role of the individual or entity
+being credited.  It must be one of the following defined credit types:
+
+- `FINDER`: identifies the vulnerability
+- `REPORTER`: notifies the vendor of the vulnerability to a CNA.
+- `ANALYST`: validates the vulnerability to ensure accuracy or severity.
+- `COORDINATOR`: facilitates the coordinated response process.
+- `REMEDIATION DEVELOPER`: prepares a code change or other remediation plans.
+- `REMEDIATION REVIEWER`: reviews vulnerability remediation plans or code changes for effectiveness and completeness.
+- `REMEDIATION VERIFIER`: tests and verifies the vulnerability or its remediation.
+- `TOOL`: names of tools used in vulnerability discovery or identification.
+- `SPONSOR`: supports the vulnerability identification or remediation activities.
+- `OTHER`
```

Comment:
Can we add a short description to this one as well?

Reply:
How is this?

> any other type or role that does not fall under the categories described above.

Reply:
SGTM👍